server/queue/queue-jms/src/test/java/org/apache/james/queue/jms/AbstractJMSMailQueueTest.java

```diff
@@ -115,6 +117,26 @@ public void testFIFO() throws Exception {
     }
 
     @Test
+    public void dequeueShouldPreserveSerializableAttributeClass() throws Exception {
+        // should be empty
+        assertEquals(0, queue.getSize());
```

Comment:
Do we agree with multiple assertions in multiple places or separate each on testing function?

Reply:
Same comment than  @mbaechler . That's some real work to sanitize this test class. Are we sure we want to do that today?

Or can we consider the fix is needed, and the refactoring of tis class is orthogonal to it?